src/olympia/devhub/templates/devhub/addons/edit.html

```diff
@@ -25,7 +25,7 @@ <h2>{{ addon.name }}</h2>
 
 <section class="primary" role="main">
 <div id="edit-addon" class="devhub-form">
-  {% if addon.current_version and addon.current_version.previews %}
+  {% if addon.current_version and addon.current_version.previews.all() %}
```

Comment:
`.exists()` would be better, to avoid evaluating the queryset needlessly

Reply:
https://github.com/mozilla/addons-server/pull/7932/commits/9bd360ff35c831448243a05b901be80dfa202548